.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
  - mvn clean install -B -q -DskipTests
  - popd
 script:
- - mvn clean install -B
+ - mvn clean verify -B
```

Comment:
Does Travis CI even still run for this project?  Might be good to port the CI stuff over to GitHub Actions

Reply:
It would be good to get rid of the custom dependency building before we do that (see https://github.com/wala/ML/blob/70d538088d6596c93325ad6d395d39a31852975a/.travis.yml#L5-L17). The current build may be found [here](https://app.travis-ci.com/github/ponder-lab/ML/builds/262805294). There's one failure do to a Python3 parsing error bug.